### ISSUE TITLE
Pilot ready branch

### ISSUE THREAD

**Issue by virgile-dev** [DemocracyEarth/sovereign#294]
Hey @paulamlb @santisiri @medied 

With @LucasIsasmendi we thought that it might be interesting to work on a pilot ready branch of Sovereign. 

The idea behing this would be to enable easy customization of Sovereign for the local context that people might use it. Lots of things are already available in the settings.json file, collective name, url etc.

Our idea is to make the `geo.json` located in`public/data` easy to adapt to a local context by allowing several data structures : 
- Universities : Oxford, Cambridge, LSE etc.
- Cities : London, Brighton etc. 
- Region : South East, West Midland etc.

What do you guys think ?

**Comment by santisiri** [comment]
Aren't we consolidating pilots on the main instance to prevent silos and harness network effects?

**Comment by paulamlb** [comment]
Yes, that's what I was thinking. Santi can you explain to non-developer me how exactly the mirrored websites work? So for example we can have venezuela.democracy.earth or stanford.democracy.earth and all of that is connected to the main instance?

**Comment by santisiri** [comment]
Yes, the simplest way is that `stanford.democracy.earth` could simply do a forward to `vote.democracy.earth/peer/stanford` and see the profile of every issue posted by the stanford user in the system. doing a change on the top level (`*.democracy.earth`) means that we'll only be able to do that with a forward.

**Comment by paulamlb** [comment]
instead of user let's do hashtag, because then anyone can publish questions - it's democratic. 

so `eleccionesvenezuela.democracy.earth` goes to `vote.democracy.earth/tag/eleccionesvenezuela`

**Comment by santisiri** [comment]
i don't like `eleccionesvenezuela` since its redundant with the word `democracy`, we need to think about how we address national debates in the platform in a fundamental way. i'm okay with doing the tag, just saying we probably want a better name (or even use a clean domain name like `eleccionesvenezuela.ve`.

I think that for countries we should have a special filter that lists proposals from people of the given country. so `vote.democracy.earth/nation/france` would list all proposals posted by french members.

**Comment by paulamlb** [comment]
`eleccionesvenezuela.ve` pointing to `vote.democracy.earth/tag/eleccionesvenezuela` ?

can we make a pinned post? with a video?

**Comment by aecc** [comment]
Slightly related topic: I'm not sure if it's a good idea to get a .ve domain. There are very unreliable, difficult to obtain, takes ages to update it in the DNS. I used to work with them and they were a mess (nic.ve is a mess like everything in Venezuela at the moment). I have heard similar opinions lately from venezuelan university professors. We probably don't want to have these kind of issues bother us after so much organizational effort :)

**Comment by AlexJupiter** [comment]
@santisiri @paulamlb 
> Aren’t we consolidating pilots on the main instance to prevent silos and harness network effects?

A couple of points on my end in terms of supporting this idea of a pilot ready branch, interested to know what you think:
- Venezuelans will be expected to be interacting with a Spanish product, a branch would allow us to create a Spanish product in a really simple way.
- After scoping out this pilot we will probably find that there will be small changes to Sovereign that will make this pilot more successful, but these changes don’t make sense to be on the main branch.
- By having branches it means that we can experiment on a micro level with the pilots, due to us being able to process the data easier, and then take these successful experimentations and eventually consolidate in the master branch.
- Building, measuring, and learning (and repeating) I think is going to be difficult to do whilst trying to harness network affects, piloted branches might help us move faster in this regard. The cross proliferation between Venezuelans and Stanford Uni students (for example) is going to be small and we might run the risk of good content getting lost in a sea of bad content with how the UX is at the moment. We will get there in time though!

**Comment by LucasIsasmendi** [comment]
@santisiri @paulamlb
> I think that for countries we should have a special filter that lists proposals from people of the given country. so vote.democracy.earth/nation/france would list all proposals posted by french members.

The concept of `~scope` could help to not only address national debates, also regional and local ones, check the [architecture-repo](https://github.com/DemocracyEarth/architecture-sovereign) for more details.
Example: scope order: ~earth -> ~france -> ~paris -> ~university|neighborhood|club|etc

@AlexJupiter 
1000% with Build, Measure, Learn cycle (we need metrics, A/B test, ranking features)

**Comment by santisiri** [comment]
@LucasIsasmendi 

> The concept of ~scope could help to not only address national debates, also regional and local ones, check the architecture-repo for more details.
Example: scope order: ~earth -> ~france -> ~paris -> ~university|neighborhood|club|etc

I really like this model. `.earth` is already the tld. Would you mean playing with the subdomain as in `france.paris.democracy.earth` ? 

I'm mindblown by the [architecture](https://github.com/DemocracyEarth/architecture-sovereign) repo. Haven't seen it until now. 

@AlexJupiter I think you have very valid points. Specially regarding language, we should include an issue for the Venezuela Milestone to include an updated translation and automatically set the language if the app is loaded for the first time via a specific URL. Regarding microexperiments, I'm completely fine on parallel implementations happening (it's open source after all), but Democracy Earth Foundation must execute with its available resources. Parallelization of ideas should happen naturally after all.
